Mathlib/Order/Filter/Basic.lean

```diff
@@ -776,6 +776,9 @@ instance unique [IsEmpty α] : Unique (Filter α) where
   uniq := filter_eq_bot_of_isEmpty
 #align filter.unique Filter.unique
 
+theorem NeBot.nonempty {f : Filter α} (hf : NeBot f) : Nonempty α :=
```

Comment:
Have you considered:
```suggestion
theorem NeBot.nonempty (f : Filter α) [f.NeBot] : Nonempty α :=
```
It's not clear to me which is preferable but if you have an application to hand, then perhaps you can tell.

Reply:
I actually have no application :facepalm: I probably used it in a previous iteration of Ascoli but don't remember how... Let's do your version then.